config/environments/development.rb

```diff
@@ -48,6 +48,9 @@
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
+  # Use a real queuing backend for Active Job (and separate queues per environment)
+  config.active_job.queue_adapter = :sidekiq
```

Comment:
Günlerini alan sorun burasıydı.  Upstream sorunu fix ettiğinde bu satır değişecek miydi?

Reply:
Bu satırı değiştirmek bize kalmış. Bu satır hiç eklenmeseydi öntanımlı adaptör `async` oluyor. Ancak `async` olursa `db:seed` patlıyor. Çünkü `async` olursa bu değer db:seed için `inline` olarak değiştiriliyor. https://github.com/rails/rails/pull/35905/files

Reply:
`async` olursa geliştirme için sidekiq'e ihtiyaç duyulmuyor. Ancak yerelde de sidekiq ile çalışmak istersek bu satırı eklemek gerekiyor.

Reply:
Aklıma gelmişken yazayım. Bir başka çözüm şuydu:

Buraya bu satırı hiç eklemeyecektik ve öntanımlı adaptör `:async` olarak kalmaya devam edecekti.
`db/seeds.rb` dosyasında ise en başa şu satırı ekleyecektik:

`ActiveJob::Base.queue_adapter = Rails.application.config.active_job.queue_adapter`